build.sbt

```diff
@@ -1,9 +1,22 @@
 import org.scalajs.linker.interface.ModuleSplitStyle
-import sbtcrossproject.CrossPlugin.autoImport.{ CrossType, crossProject }
+import sbtcrossproject.CrossPlugin.autoImport.{CrossType, crossProject}
 import sbt._
 import sbt.io.Using
 
-val scalaVer                = "3.1.0"
+val versions: Map[String, String] = {
```

Comment:
I wonder if it is simpler to use `sbt-github-actions` doing it the other way around, generate the yml from the sbt

Reply:
Personally I find this _way_ simpler. There's just one source of truth: the GitHub Action config. No need for any manual code generation at all.